phenotools-builder/src/main/java/org/phenopackets/phenotools/builder/builders/PedigreeBuilder.java

```diff
@@ -0,0 +1,25 @@
+package org.phenopackets.phenotools.builder.builders;
+
+import org.phenopackets.schema.v2.core.Pedigree;
+
+public class PedigreeBuilder {
+
+    private final Pedigree.Builder builder;
+
+    public PedigreeBuilder() {
+        builder = Pedigree.newBuilder();
+    }
+
+    public PedigreeBuilder person(Pedigree.Person person) {
```

Comment:
Looking at `FamilyWithPedigree` example, it may be good to add methods with the following signatures: 
- `public PedigreeBuilder addAllPersons(Pedigree.Person... persons)`
- `public PedigreeBuilder addAllPersons(Collection<Pedigree.Person> persons)`

Reply:
Good idea, added
 public PedigreeBuilder addAllPersons(Pedigree.Person... persons) {
        builder.addAllPersons(Arrays.asList(persons));
        return this;
    }

    public PedigreeBuilder addAllPersons(Collection<Pedigree.Person> persons){
        builder.addAllPersons(persons);
        return this;
    }